src/main.ts

```diff
@@ -685,11 +703,15 @@ function getFilePathFromName(name: string) {
   }
 }
 
-const stopProxyProcess = () => {
+const stopProxyProcess = async () => {
   if (currentProxyProcess) {
-    // NOTE: this might not kill the second spawned process on windows
     currentProxyProcess.kill()
     currentProxyProcess = null
+
+    // kill remaining proxies if any, this might happen on windows
+    if (getPlatform() === 'win') {
+      await cleanUpProxies()
+    }
```

Comment:
Ensures that other instances of the proxy are killed. This is the same logic used during launch